### ISSUE TITLE
Popup window resize

### ISSUE THREAD

**Issue by x20mar** [novnc/noVNC#144]
I thought I should check first if this feature exists or not before I write some code to resolve this issue.

I've been asked to change the way I was opening up the vnc page from just opening a new window to a pop up window (done in javascript).

Now while I can do this no bother I've been asked to resize the vnc window according to the canvas size and status bar height. Having been looking at your code for a while I guess I need to write a callback function to implement this correctly.

But before I did that I thought I should check with you if that's the best way to resolve this.

tl:dr if once loaded canvas size is 720 x 400, resize window to be 720 x 435
